src/index.ts

```diff
@@ -31,6 +31,7 @@ function containsColumn(c: PgConstraint, attr: PgAttribute): boolean {
 function getNodeIdRelations(table: PgClass, build: any) {
   return table.constraints
     .filter(isForeignKey)
+    .filter(c => !build.pgOmit(c, "filter"))
```

Comment:
@benjie I don't think this is quite right (need to check different permissions in different cases), but we definitely need to respect `@omit`.